src/webapp/pages/import-template/ImportTemplatePage.tsx

```diff
@@ -329,8 +329,10 @@ export default function ImportTemplatePage({ settings }: RouteComponentProps) {
                                 </React.Fragment>
                             ) : (
                                 <React.Fragment>
-                                    {moment(String(group.period)).format("DD/MM/YYYY")}:{" "}
-                                    {group.id ? i18n.t("Update") : i18n.t("Create")} {group.count}{" "}
+                                    {moment(String(group.period)).format(
+                                        importState.dataForm.periodType === "Yearly" ? "YYYY" : "DD/MM/YYYY"
```

Comment:
Do we want this change in the PR?

Reply:
Yes, inside the issue this is related to the task "In Bulk Load summary when importing data values of a custom template, each year appears as a date (e.g. 1/1/2022) instead of a period (2022) even if the dataset is a yearly dataset"